crates/uv/src/commands/project/sync.rs

```diff
@@ -297,6 +285,23 @@ pub(super) async fn do_sync(
     Ok(())
 }
 
+fn apply_install_options(
```

Comment:
Should this be `InstallOptions::filter_resolution(...)` or something? (and the helpers would all become private members?)